include/z64player.h

```diff
@@ -1144,8 +1154,8 @@ typedef struct Player {
     /* 0xB67 */ u8 remainingHopsCounter; // Deku hopping on water
     /* 0xB68 */ s16 fallStartHeight; // last truncated Y position before falling
     /* 0xB6A */ s16 fallDistance; // truncated Y distance the player has fallen so far (positive is down)
-    /* 0xB6C */ s16 unk_B6C;
-    /* 0xB6E */ s16 unk_B6E;
+    /* 0xB6C */ s16 floorPitch; // angle of the floor slope in the direction of current world yaw (positive for ascending slope)
+    /* 0xB6E */ s16 floorPitchAlt; // the calculation for this value is bugged and doesn't represent anything meaningful
```

Comment:
could you explain why it's considered bugged?

Reply:
Do you mean explain it in this thread or explain in the comment/where it's calculated?
Here's the start of the message thread working it out:
https://discord.com/channels/688807550715560050/838852326231769089/1064327449690771497